src/backend/api/Fusion.Resources.Api/Startup.cs

```diff
@@ -128,33 +121,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
 
-                // TODO: Remove
```

Comment:
ChaosMonkey to be replaced with nuget package.

Reply:
yepp. Should add this at a later time